core/mobile-manager/src/backend/iOSHost.ts

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { ClientRequestContext } from "@bentley/bentleyjs-core";
+import { IModelHostConfiguration, IpcHostOptions } from "@bentley/imodeljs-backend";
+import { IpcWebSocketBackend } from "@bentley/imodeljs-common";
+import { BatteryState, Orientation } from "../common/MobileAppProps";
+import { DownloadTask, MobileCompletionCallback, MobileDevice, MobileDeviceAuthSettings, MobileHost, MobileProgressCallback } from "./MobileHost";
+
+type DeviceEvents = "memoryWarning" | "orientationChanged" | "enterForeground" | "enterBackground" | "willTerminate";
+
+/** @beta */
+export class IOSDevice extends MobileDevice {
+  public emit(eventName: DeviceEvents, ...args: any[]) {

```

Comment:
why not put this method on MobileDevice?

Reply:
It's only called from the native side. I guess we can -- no reason why the android native side would need to be different.